app/javascript/articles/Article.jsx

```diff
@@ -49,10 +50,14 @@ export const Article = ({
         role="presentation"
         onClick={(event) => {
           const { classList } = event.target;
-
           if (clickableClassList.includes(...classList)) {
-            InstantClick.preload(article.path);
-            InstantClick.display(article.path);
+            if (event.which > 1 || event.metaKey || event.ctrlKey) { // Indicates should open in _blank
+              window.open(article.path, '_blank');
```

Comment:
Given that this will only link to other content _within_ the platform it shouldn't really matter—If anything there _could_ be a legit purpose for possibly having access to the prior window for any reason.